obspy/core/tests/test_stream.py

```diff
@@ -1736,11 +1736,11 @@ def test_copy(self):
         """
         st = read()
         st2 = st.copy()
-        self.assertTrue(st == st2)
-        self.assertTrue(st2 == st)
+        self.assertEqual(st, st2)
+        self.assertEqual(st2, st)
         self.assertFalse(st is st2)
```

Comment:
And I didn't even look at the assertFalse yet.

These are very, very far from covering all assertTrue and all assertFalse. However, they should be a step in the correct direction. Idea is to get more informative log messages, such that we ideally do not even need to rerun the test or debug for seeing what's happening.